src/CMakeLists.txt

```diff
@@ -163,8 +183,8 @@ if(DEBUG_USE_ASAN AND DEFINED DEBUG_BUILD)
 	CHECK_AND_SET_FLAGS(COMPILER_SUPPORTS_ASAN "-fsanitize=address" DEBUG_FLAGS "-fsanitize=address")
 endif()
 
-if(RELEASE_USE_LTO)
-	set(RELEASE_FLAGS "${RELEASE_FLAGS} -flto")
+if(RELEASE_USE_LTO AND NOT DEFINED BEBUG_BUILD)
```

Comment:
* There's a typo here (**B**EBUG_BUILD), and LTO will be enabled for any build type, consequently
* `DEBUG_BUILD` isn't appropriate here in the first place, because it's also set for `RelWithDebInfo`. We want to have LTO available on both `Release` and `RelWithDebInfo` builds. There is a definition of `RELEASE_BUILD` down below, I suggest to move it into the global scope near `DEBUG_BUILD`, and use it in this test.

Reply:
Yes, looks like it is better way to handle this. I'll fix it too.